src/plopp/functions/slicer.py

```diff
@@ -105,7 +105,8 @@ def __init__(
         self.slice_nodes = [
             slice_dims(data_node, self.slider_node) for data_node in self.data_nodes
         ]
-        if len(keep) == 1:
+        ndims = len(keep)
+        if ndims == 1:
```

Comment:
Both branches pass the same args to `figure1d` and `figure2d`. You can avoid this, e.g., by using
```python
if ndims == 1:
    make_figure = figure1d
# ...
self.figure = make_figure(...)
```
or something similar.

Reply:
I like minor comments ;-)